src/core/lib/gpr/tls_stdcpp.h

```diff
@@ -0,0 +1,48 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPC_CORE_LIB_GPR_TLS_STDCPP_H
+#define GRPC_CORE_LIB_GPR_TLS_STDCPP_H
+
+/** Thread local storage based on C++11 standard primitives.
+#include <grpc/support/port_platform.h>
```

Comment:
This line should be outside the comment?

Reply:
I made this deliberately because of `tls_msvc.h` which did the same thing but `tls_pthread.h` and `tls_gcc.h` which did the opposite. I guess either way would be fine because they're included as an inline header only from `tls.h` which includes `grpc/support/port_platform.h`. But I like to put this outside so that it looks like others.